lib/jnpr/junos/facts/routing_engines.py

```diff
@@ -38,7 +41,7 @@ def facts_routing_engines(junos, facts):
                 re_fd[factoid.replace('-', '_')] = x_f.text
 
         if 'mastership_state' in re_fd:
-            if facts[re_name]['mastership_state'] == 'master':
+            if facts[re_name]['mastership_state'].find('master') > 0:
```

Comment:
@dgjnpr Can you please explain the reason for this changes. There are most chances of having mastership_state value to be 'master' itself. In that case find will give 0 and > 0 will fail.

```python
>>> facts['RE0']['mastership_state']
'master'
>>> facts['RE0']['mastership_state'].find('master')
0
>>> facts['RE0']['mastership_state'].find('master')>0
False```